XenAdmin/Dialogs/AssignLicenseDialog.cs

```diff
@@ -99,25 +99,39 @@ private List<Host> CreateHostsList()
             return hosts;
         }
 
-
-        /// <summary>
-        /// Not all license types apply for post-clearwater hosts. Hide them to declutter the UI
-        /// </summary>
-        private void SetOptionsForClearwaterAndNewer()
+        private void LoadLicenseOptions()
         {
-            if (xos.TrueForAll(x => Helpers.CreedenceOrGreater(x.Connection)))
+            if (xos.TrueForAll(x => Helpers.NaplesOrGreater(x.Connection)))
             {
                 perSocketRadioButton.Visible = false;
                 xenDesktopEnterpriseRadioButton.Visible = false;
                 enterprisePerSocketRadioButton.Checked = true;
-                enterprisePerSocketRadioButton.Text = String.Format(Messages.ENTERPRISE_PERSOCKET_LICENSES_X_REQUIRED,
-                                                          xos.Sum(x => x.Connection.Cache.Hosts.Sum(h => h.CpuSockets())));
-                standardPerSocketRadioButton.Text = String.Format(Messages.STANDARD_PERSOCKET_LICENSES_X_REQUIRED,
-                                                          xos.Sum(x => x.Connection.Cache.Hosts.Sum(h => h.CpuSockets())));
+                enterprisePerSocketRadioButton.Text = String.Format(Messages.LICENSE_EDITION_ENTERPRISE_PERSOCKET, 
+                    xos.Sum(x => x.Connection.Cache.Hosts.Sum(h => h.CpuSockets()))); 
+                enterprisePerUserRadioButton.Text = Messages.LICENSE_EDITION_ENTERPRISE_PERUSER;
+                desktopPlusRadioButton.Text = Messages.LICENSE_EDITION_DESKTOP_PLUS;
+                desktopRadioButton.Text = Messages.LICENSE_EDITION_DESKTOP;
+                desktopCloudRadioButton.Text = Messages.LICENSE_EDITION_DESKTOP_CLOUD;

```

Comment:
`desktopCloudRadioButton` should be made visible here (as it is not visible by default)

Reply:
Agree. And I found enterprisePerUserRadioButton had visibility problem too. Going to fix it.